modules/tinymce/src/core/test/ts/browser/selection/RangeWalkTest.ts

```diff
@@ -0,0 +1,113 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyHooks, TinySelections } from '@ephox/mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import * as RangeWalk from 'tinymce/core/selection/RangeWalk';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.core.selection.RangeWalkTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme ]);
+
+  const complexList = '<ul>' +
+    '<li>Test 1</li>' +
+    '<li>Test 2' +
+      '<ul>' +
+        '<li>Test 2.1</li>' +
+        '<li><strong>Test 2.2</strong>' +
+          '<ul>' +
+            '<li>Test 2.2.1</li>' +
+          '</ul>' +
+        '</li>' +
+      '</ul>' +
+    '</li>' +
+    '<li>Test 3</li>' +
+  '</ul>';
+
+  const testWalking = (editor: Editor, expected: string[][]) => {
+    const nodeNames: string[][] = [];
+
+    const rng = editor.selection.getRng();
+    RangeWalk.walk(editor.dom, rng, (nodes) => {
+      nodeNames.push(Arr.map(nodes, (node) => node.nodeName.toLowerCase()));
+    });
+
+    assert.deepEqual(nodeNames, expected);
+  };
+
+  it('TBD: should only include itself if the full node is selected', () => {
```

Comment:
These are `TBD`s as it's not applicable to this change and just tests the existing functionality.

Reply:
Elsewhere I think we use `TBA`, but either way didn't we agree we would stop adding those for tests that we didn't know what Jira they were associated with?

Reply:
You're correct on both accounts... I shall remove them.

- [ ] Remove `TBD` prefixes

Reply:
Fixed